server/QuestionsAndAnswers/answers.js

```diff
@@ -0,0 +1,20 @@
+var routes = require('express').Router();
+var axios = require('axios');
+var url = require('url');
+var config = require('../../config');
+var endPoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/:hr-atx/qa/answers'
+
+var auth = { headers: { 'Authorization': `${config}` }, params: { product_id: 1 } };
+
+routes.put(`/:answer_id/*`, (req, res) => {
+  var path = url.parse(req.url, true).path.split('/').at(-1);
```

Comment:
can be `const`. also, what happens if there is nothing to split `.at(-1)`?

Reply:
Yes const is a better key word. 
Hmm, it seems i didn't account for the possibility of there being nothing to split. I have to account for that.